debian/postinst

```diff
@@ -22,6 +22,31 @@ case $1 in
 	systemctl enable systemd-networkd.service
 
 	systemctl disable walinuxagent.service
+
+	if ! id -u postgres > /dev/null; then
+		# When installing postgres, a postgres user is created unless it
+		# already exists. To have a consistent UID accross installations
+		# and to facilitate migration from Illumos we create a postgres
+		# user with the same UID as in Illumos. The Ubuntu postgres
+		# package requires that the postgres user is also part of the
+		# postgres group. Again, for consistency, we create the postgres
+		# group with a pre-determined GID, which the same as the
+		# postgres UID.
+		addgroup postgres --gid 65437
+		adduser --home /var/lib/postgresql --no-create-home \
+		    --shell /bin/bash --ingroup postgres \
+		    --gecos "PostgreSQL administrator" --uid 65437 \
+		    --disabled-password postgres
+		# To facilitate migration, we also add postgres to group 10,
+		# which is the GID of group staff on Illumos. domain0/mds is
+		# owned by delphix:staff on Illumos.
+		usermod -a -G 10 postgres
+		# Finally, we also add postgres to the Linux staff group, so
+		# that it can access files created by the delphix user.
+		usermod -a -G staff postgres
+	elif [[ $(id -u postgres) -ne 65437 ]]; then
+		echo "ERROR: postgres id $(id -u postgres) != 65437" >&2
```

Comment:
this message says "ERROR", but we don't actually return an error.. so package installation/upgrade will report "success".. is this intentional?

Reply:
Good catch, I'll fix this.